test/tests/functional/pbs_power_provisioning_sgi.py

```diff
@@ -72,7 +120,6 @@ def setup_sgi_api(self, script, perm=0755):
         os.write(fd, script)
         os.close(fd)
         os.chmod(fn, perm)
```

Comment:
use dshutils chmod rather than using os directly

Reply:
I have addressed this comment.